src/components/attendee/VideoComms/scss/VideoCommsParticipant.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "scss/constants";
+
+.videoCommsParticipant {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  height: 100%;
+  max-width: $video-participant-optimal-width;
+  min-width: $video-participant-optimal-width;
+  max-height: $video-participant-optimal-height;
+  width: 100%;
+  background: #3d3a3f;
+  border-radius: $border-radius--xl;
+  overflow: hidden;
+}
+
+.trackContainer {
+  display: flex;
+  position: relative;
+  width: 100%;
+  video {
+    object-fit: cover;
+    width: 100%;
+  }
+}
+
+.videoCommsControlsContainer {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  flex-direction: row-reverse;
+  font-size: 22px;
+  color: #f1f1f1;
```

Comment:
might be worthwhile to add some hardcoded values to our `attendee` variables and reuse from there